scripts/pvscheck.sh

```diff
@@ -343,13 +343,15 @@ run_analysis() {(
 
   cd "$tgt"
 
+  # pvs-studio-analyzer exits with a non-zero exit code when there are detected
+  # errors, so ignore its return
   pvs-studio-analyzer \
     analyze \
       --threads "$(get_jobs_num)" \
       --output-file PVS-studio.log \
       --verbose \
       --file build/compile_commands.json \
-      --sourcetree-root .
+      --sourcetree-root . || true
```

Comment:
will this make it more difficult to notice a problem with the bot-ci job?

Reply:
The bot-ci job has been failing for the past two weeks because of the non-zero exit code.  The only thing this line does is run the analyzer.  Whether that fails or not, there isn't anything we can do about it.  However, the failure is preventing us from generating the reports that we post to neovim.io, so we don't lose any useful information and we gain the actual completion of the bot-ci job.